.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -0,0 +1,45 @@
+# .NET Core Release Checklist
+
+## Release Versions
+
+_The set of .NET Core versions that are being released as a unit._
+
+* 2.1.X: <runtime/SDK>
+* 3.1.X: <runtime/SDK>
+
+1. - [ ] Update target branch to new version.
+   - For 2.x:
+     - [ ] If releasing 2.x, run auto-update (refer to docs).
```

Comment:
> refer to docs

Are there links missing where this phrase shows up in the template?

Reply:
Didn't want to put A&D One Note links here as the general audience may start requesting access in large numbers.

Reply:
Maybe then "refer to internal Microsoft docs"? Or else people might start commenting on the release issues themselves asking what docs are being referred to. 😛

If this were for a broader set of people to use, I'd want to include at least the name of the OneNote and the path to the applicable page, but since source-build is a small team I suppose it is fine to have a general statement.